packages/react-scripts/template/src/index.css

```diff
@@ -1,5 +1,12 @@
+* {
+  box-sizing: border-box;
```

Comment:
FWIW, I've seen normalize used on a lot of projects. It doesn't handle box sizing, so I typically refer to Paul Irish's [canonical source](https://www.paulirish.com/2012/box-sizing-border-box-ftw/).